config/rhoai/webhook_network_policy.yaml

```
@@ -0,0 +1,14 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: kueue-webhook-server
```

```suggestion
  name: webhook-server
```
Kustomize attach `kueue` prefix

---

Good catch, thanks! Updated.